### ISSUE TITLE
Enhance visualization of object detection result

### ISSUE THREAD

**Issue by wkentaro** [start-jsk/jsk_apc#1058]
### Enhance (A) to (B)

(A)
<img width="300" src="https://cloud.githubusercontent.com/assets/4310419/12951844/ab795cce-d057-11e5-9d07-3018ac477dc7.png" />

(B)
<img width="300" alt="screen shot 2016-02-11 at 12 38 33 am" src="https://cloud.githubusercontent.com/assets/4310419/12951898/d1dee960-d057-11e5-957e-90cbbfd45836.png">
<img width="300" alt="screen shot 2016-02-11 at 12 39 06 am" src="https://cloud.githubusercontent.com/assets/4310419/12951912/e30a8668-d057-11e5-9990-6fdba1d7a307.png">

sample:

- https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_perception/node_scripts/rect_array_to_image_marker.py
- https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_perception/node_scripts/fast_rcnn_caffenet.py#L52

but maybe we need to keep the region information of label image for semantic object detection result.
@knorth55

**Comment by knorth55** [comment]
I tried to use jsk_perception/BoundingBoxToRect to visualize bounding boxes, but somehow I couldn't get any output.
I checked subscribing topics (BoundingBoxArray and CameraInfo) are ok

```bash
$ roslaunch jsk_2016_01_baxter_apc kinect2_external.launch
$ roslaunch jsk_2016_01_baxter_apc recognition_in_stow.launch
$ roslaunch jsk_2016_01_baxter_apc viz_recognition.launch INPUT_BOXES:=/stow_cpi_decomposer/boxes CAMERA_INFO:=/kinect2_external/hd/camera_info
```

https://github.com/knorth55/jsk_apc/blob/enh-viz/jsk_2016_01_baxter_apc/launch/include/viz_recognition.launch

**Comment by knorth55** [comment]
WIP
- [ ] Use jsk_perception/BoundingBoxToRect to modify Bounding Boxes to Rectangles
- [ ] Draw rectangles and recognition results on label image

**Comment by wkentaro** [comment]
Closed via #1241
